tools/run_tests/jobset.py

```diff
@@ -224,20 +213,19 @@ def start(self):
 
   def state(self, update_cache):
     """Poll current state of the job. Prints messages at completion."""
+    self._tempfile.seek(0)
+    stdout = self._tempfile.read()
+    # TODO: looks like jenkins master is slow because parsing the junit 
+    # results XMLs is not implemented efficiently. This is an experiment to 
+    # workaround the issue by making sure results.xml file is small enough.
+    if len(stdout) > _MAX_RESULT_SIZE:
```

Comment:
wouldn't just self.result.message = stdout[-_MAX_RESULT_SIZE:] without  the if - else suffice?